frontend/src/Services/api.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+const DEVELOPMENT_URL = 'http://localhost:4000/api/';
+
+export const axiosInstance = axios.create({
+    baseURL: DEVELOPMENT_URL
```

Comment:
This solution won't work when we deploy it to AWS. We should replace this into environment variable